nio.json

```diff
@@ -0,0 +1,182 @@
+{
+  "non_editable": {             // editable @ admin/owner level?
```

Comment:
I wouldn't segregate between non-editable and editable.  I'd eventually put the onus of this on the module that's manipulating this structure.  Since edit-ability could depend on where the instance is running.